### ISSUE TITLE
TEI to JSON conversion gives strange behavior with <pc/> inside <supplied/>

### ISSUE THREAD

**Issue by jjmccollum** [d-flood/Tendon#3]
Here's another issue that is probably due more to idiosyncrasies in the ITSEE transcription tool than to Criticus but is still worth mentioning: some IGNTP transcriptions (I've noticed this in particular in the Ephesians transcription for 025) place a punctuation mark (element `<pc/>`) inside a `<supplied/>` element within a `<w/>`. This is probably not how this should be done (I would probably place the `<pc/>` parallel to the `<w/>` elements and include the `<supplied/>` tag as a child of it, which is the practice for words), and the fact that the transcription tool adds a `<w/>` element with an empty `<supplied/>` child afterwards confirms that something's wrong, but the TEI guidelines technically allow this, so everything passes validation.

The issue at hand is how Criticus handles this when it converts the TEI XML to JSON. In the JSON output, the punctuation token is treated as it own word, and there is a hanging left bracket with no matching right bracket. This should be addressed, as it causes the ITSEE collation editor to fail when it passes the JSON to CollateX.

**Comment by d-flood** [comment]
I've looked at 025 and a few others to see how the `<pc/>` elements are being used. It looks like it will be safe to entirely strip them and their child elements/text. I couldn't find any examples of `<pc/` elements enclosing anything other than a single punctuation mark.

I've added ` text = re.sub(r'<pc>[^<>]+</pc>', '', text)` to `from_tei.py`. I think that will work. If you know of a specific verse in 025 in which a punctuation was being treated as a word, I'll give it a test.

To fix any potentially hanging left bracket, I've also added `text = re.sub(r' *<supplied[^<>]*/>', '', text)` to `from_tei.py`.

I found an example exactly as you described at Eph 2:3 (B10K2V3) in 025. The JSON output included a punctuation mark as a word and even a left bracket as a word. Both of these have been resolved.

Thank you so much for the well-described issues and suggested fixes. It made it easy for me to get started.
